media/CircleEditor/CircleEditor.md

```diff
@@ -0,0 +1,9 @@
+## Circle Editor
```

Comment:
`README.md` is better as file name :)
And my intention was following :sweat_smile: 

```
## Circle Editor

This extension adds modifiable circle editor, which provides graphical viewer and editor for `*.circle` files.
```